sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.netstandard2.0.cs

```diff
@@ -9,6 +9,111 @@ public partial class BinaryContent
         public virtual System.BinaryData? Data { get { throw null; } set { } }
         public virtual bool IsReadOnly { get { throw null; } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct Value
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public Value(System.ArraySegment<byte> segment) { throw null; }
```

Comment:
Why ArraySegment and not Span?

Reply:
It's not possible to fish out the array/memory out of spans. the only way to get access to it would be via copy (which is expensive and allocating)